### ISSUE TITLE
Yosemite?

### ISSUE THREAD

**Issue by hoondi** [ysalathe/GenericUSBMIDI#8]
Hi,

Just curious if this works for Yosemite?

**Comment by hoondi** [comment]
yes it does!

Confirmed on 10.10.4, although a lot of deprecation warnings with Xcode 6.4...

**Comment by anandaroop** [comment]
Confirmed on 10.10.5. Thanks Yves!

**Comment by nikocena** [comment]
does this work in el capitan?

**Comment by ReijoDee** [comment]
Works on El Capitan 10.11.6

**Comment by nikocena** [comment]
Yes, it worked. I kinda took the risk of compiling it and used it for my obsolete Roland Multitimbral Synthesizer.

**Comment by masanaohayashi** [comment]
Thank you for the code. I just confirmed it works on Sierra. I changed the code to work with UM-1, UM-2, and UM-3/UM-3EX with one driver file. BTW I think it's good to change the third argument of MIDIDeviceAddEntity() to false to enable ports to be connected to external device. By this change, you can see I/O ports in Audio MIDI Settings.

Followings are product id, # ports and configuration descriptor to use.
UM-1: 0x0008, 1, 2
UM-2: 0x0005, 2, 2
UM-3: 0x009a, 3, 0

**Comment by dspasoski** [comment]
Hey guys!

Would someone please provide a built driver for Yosemite?

Thanks!
